### ISSUE TITLE
Doesn't work for Google Drive and Dropbox links

### ISSUE THREAD

**Issue by RajChanchal** [mwaterfall/MWPhotoBrowser#366]
Hi,
Let me know how to use it for photoes with shareable links from dropbox and google drive? It doesn't load at the moment.

**Comment by fparkar** [comment]
what is the link you are trying? could you share the link here please?

**Comment by RajChanchal** [comment]
Check, for example, this link to a picture: https://db.tt/uzkJkA1u
We can open it in a browser though.

**Comment by mythodeia** [comment]
this link will not work because the response is not an image. its an html containing an image.
the response headers should be image/jpeg or image/png

**Comment by RajChanchal** [comment]
Thanks, but what about the google drive link, for example: https://docs.google.com/uc?id=0B_1hGRxJtrLjZkZDTG5lLU5VcU0&export=download
Is it also an HTML link? It is also not loading.. :(

**Comment by mythodeia** [comment]
the link you provided is 403 error 
can you check if its valid on your browser?

**Comment by RajChanchal** [comment]
Yes, it is a valid link. And clicking on the link directly starts downloading the image file from google drive. I tried to open the above link in another browser too (since I thought of running session of google drive on the current browser) and it did also download the file!!

**Comment by RajChanchal** [comment]
This is the log printed in the gallery for the file: 
Printing description of attachementModel->_downloadUrl:
https://docs.google.com/uc?id=0B_1hGRxJtrLjZkZDTG5lLU5VcU0&export=download
It seems to try loading the file (it shows loader), but shows the cross icon at the end.. :/

**Comment by fparkar** [comment]
logout from gmail account and then open the link... it asks for username & password... such links wont work... it should be direct path to image...

**Comment by mythodeia** [comment]
if you open the url you provided on the browser you will see that the error states
``` 403 error We're sorry, but you do not have access to this page. ```

**Comment by bpoplauschi** [comment]
Closing, this is not an issue with SDWebImage.
